Energy Web/energy-web-mainnet-starter/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT LICENSE
+
+Copyright 2020-2021 SubQuery Pte Ltd authors & contributors
```

Comment:
Where is this template from? It should say 2020-2024?

Reply:
Ok, have fixed https://github.com/subquery/ethereum-subql-starter/commit/b9ea15a0c538bba6def463ffcc84f1112146510a, but can you update the template please?